components/Page.js

```diff
@@ -42,7 +43,6 @@ class Page extends React.Component {
             @media (min-width: 1024px) {
               .main {
                 ${COLUMN};
-                min-width: 1080px; /* temporary fix for mobile overflow issue */
```

Comment:
@aquaductape does this line have to be removed? I can't remember why, but I intentionally included it here: https://github.com/carbon-app/carbon/commit/10cc6fdb935437a9e153c4b275198780644543e3#diff-2465a0749a2041e522b6b8d7681adf5aR31-R37

What do you think?

Reply:
It was probably there when it depended on a different layout. After removing it, I made sure the layout didn't break throughout the viewport width. I also checked mobile devices viewport on chrome(even though it's not 100% accurate) and on my galaxy s10e and didn't see any breaking layout.